confignode/src/main/java/org/apache/iotdb/confignode/persistence/NodeInfo.java

```diff
@@ -93,7 +85,7 @@ public class NodeInfo implements SnapshotProcessor {
 
   // Online DataNodes
   private final ReentrantReadWriteLock dataNodeInfoReadWriteLock;
-  private final AtomicInteger nextNodeId = new AtomicInteger(1);
+  private final AtomicInteger nextNodeId = new AtomicInteger(0);
```

Comment:
why is 0?   if isseednode , nodeid is 0.

Reply:
The Seed-ConfigNode will apply itself